scripts/partition_manager/partition_manager.rst

```diff
@@ -0,0 +1,408 @@
+.. _partition_manager:
+
+Partition Manager
+#################
+
+Partition Manager is a Python script that sets the start address and size of all image partitions in a multi-image build context.
+When creating an application that requires child images (for example, a bootloader), you can use Partition Manager to control where in memory each image should be placed.
```

Comment:
But it isn't used unless you decide you want it, and you create the config files. ;)
Changed to "you can configure Partition Manager"

Reply:
Note that if you enable existing sub-images (mcuboot f.ex.) Partition Manager will be used automatically since there is already a pm.yml.

The change is 👍